test-loop-tests/src/tests/cloud_archival.rs

```diff
@@ -0,0 +1,82 @@
+use std::collections::HashSet;
+
+use near_async::time::Duration;
+use near_chain_configs::test_genesis::{TestEpochConfigBuilder, ValidatorsSpec};
+use near_chain_configs::test_utils::test_cloud_archival_configs;
+use near_o11y::testonly::init_test_logger;
+use near_primitives::shard_layout::ShardLayout;
+use near_primitives::types::AccountId;
+
+use crate::setup::builder::TestLoopBuilder;
+
+const EPOCH_LENGTH: u64 = 10;
+const GC_NUM_EPOCHS_TO_KEEP: u64 = 3;
+
+#[test]
+fn test_cloud_archival_base() {
```

Comment:
Nit: consider adding a comment to say what subset of cloud archival features this test is checking